worker/lib/workers/qemu.ts

```diff
@@ -188,6 +205,12 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 		});
 	}
 
+	private async iptablesRules(bridgeName: string, bridgeAddress: string){
+		await execProm(`iptables-legacy -I FORWARD -i ${bridgeName} -j ACCEPT -m comment --comment ${this.iptablesComment}`);
+		await execProm(`iptables-legacy -I FORWARD -o ${bridgeName} -m state --state RELATED,ESTABLISHED -j ACCEPT -m comment --comment ${this.iptablesComment}`);
+		await execProm(`iptables-legacy -t nat -A POSTROUTING ! -o ${bridgeName} --source ${bridgeAddress}/24 -j MASQUERADE -m comment --comment ${this.iptablesComment}`);
```

Comment:
Need to test this command on real jenkins-agent shell in balena-jenkins fleet.

https://dashboard.balena-cloud.com/fleets/1841113/summary

Reply:
works within jenkins jobs because the container is using host mode networking with the NET_ADMIN capability